.github/workflows/security.yml

```
@@ -0,0 +1,456 @@
+
+      - name: Install JavaScript dependencies
+        if: matrix.language == 'javascript'
+        working-directory: apps/web
+        run: |
+          npm ci --prefer-offline --no-audit
+
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v3
+        with:
+          category: "/language:${{matrix.language}}"
+          fail-on-error: false
```

actionlint (1.7.7)

input "fail-on-error" is not defined in action "github/codeql-action/analyze@v3". available inputs are "add-snippets", "category", "check_name", "checkout_path", "cleanup-level", "expect-error", "matrix", "output", "ram", "ref", "sha", "skip-queries", "threads", "token", "upload", "upload-database", "wait-for-processing"

(action)